### DISCUSSION TITLE
Jupyter Lab not opening more than one file

### DISCUSSION THREAD

**Discussion by ArijitSinghEDA** [jupyterlab/jupyterlab#15359] [Q&A] [answered]
I am using jupyter lab, and when I try to open an `.ipynb` file, I get the following warning:
![jl2](https://github.com/jupyterlab/jupyterlab/assets/132063833/3dfb7f28-b029-4bd4-b9cd-8ef4e779e98e)

I have closed and opened my jupyter lab multiple times, I have even upgraded from v4.0,4 to v4.0.8. Anyone who can help me this?

P.S. I have not used `--collaborative` flag while starting my Jupyter lab, but it is still showing RTC when it opens in my browser.

**Comment by krassowski** [comment, marked as answer]
To disable RTC in 4.0 you need to uninstall `jupyter_collaboration` - the --collabrative flag is no-op. The configuration is handled on the other side too: https://jupyterlab-realtime-collaboration.readthedocs.io/en/latest/configuration.html
